extensions/extensions-docker/aissemble-spark/src/main/resources/docker/Dockerfile

```diff
@@ -37,6 +37,8 @@ RUN apt-get update -y && apt-get install --assume-yes \
   && apt-get clean \
   && ln -s /usr/bin/python${PYTHON_VERSION} /usr/bin/python
 
+RUN ln -sf /usr/bin/python3.11 /usr/bin/python3
```

Comment:
A: Let's put a comment explaining that spark uses `python3` to execute pyspark pipelines.